sidebars.js

```diff
@@ -30,14 +30,10 @@ module.exports = {
       items: [
 	'graphql',
         {
-	  GraphQL:['gql-introduction','gql-queries','gql-mutations','gql-directives','gql-interfaces','gql-schema-types'],
-	},
-	{
-	  'GraphQL CRUD Tutorial': ['gql-crud-tutorial'],
-	}, 
-	{
-	  'GraphQL Time Series Aggregation': ['time-series-aggregations'],
+	  GraphQL Features: ['gql-introduction','gql-queries','gql-mutations','gql-directives','gql-interfaces','gql-schema-types'],
```

Comment:
Keys must be in quotes. `GraphQL Features` must be `'GraphQL Features'`

Reply:
Added